File: Valghalla.Internal.Web/src/features/administration/task-type/models/update-task-type-request.ts
export interface UpdateTaskTypeRequest {
  id: string;
  title: string;
  shortName: string;
  description: string;
  startTime: string;
  endTime: string;
  payment?: number;
  validationNotRequired: boolean;
  trusted: boolean;
  sendingReminderEnabled: boolean;
  fileReferenceIds: string[];
  newWorkLocationId?: string;
  workLocationId?: string;
  newElectionId?: string;
  electionId?: string;
  newTaskTypeTemplateId?: string;
  taskTypeTemplateId?: string;
}

File: Valghalla.Internal.API/Requests/Administration/TaskType/UpdateTaskTypeRequest.cs
namespace Valghalla.Internal.API.Requests.Administration.TaskType
{
    public class UpdateTaskTypeRequest
    {
        public Guid Id { get; init; }
        public string Title { get; init; } = null!;
        public string ShortName { get; init; } = null!;
        public string Description { get; init; } = null!;
        public TimeSpan StartTime { get; init; }
        public TimeSpan EndTime { get; init; }
        public int? Payment { get; init; }
        public bool ValidationNotRequired { get; init; }
        public bool Trusted { get; init; }
        public bool SendingReminderEnabled { get; init; } = true;
        public IEnumerable<Guid> FileReferenceIds { get; init; } = Enumerable.Empty<Guid>();
        public Guid NewElectionId { get; set; }
        public Guid ElectionId { get; set; }
        public Guid NewWorkLocationId { get; set; }
        public Guid WorkLocationId { get; set; }
        public Guid NewTaskTypeTemplateId { get; set; }
        public Guid TaskTypeTemplateId { get; set; }

    }
}
